nav2_bringup/launch/localization_launch.py

```diff
@@ -40,6 +40,7 @@ def generate_launch_description():
     params_file = LaunchConfiguration('params_file')
     use_composition = LaunchConfiguration('use_composition')
     container_name = LaunchConfiguration('container_name')
+    container_name_full = (namespace, '/', container_name)
```

Comment:
Why not have the bringup file have the `namespace/container_name` as the field for `namespace=` so that the children launch files (localization / navigation) don't have to think about it?

Reply:
That was initial solution, but I've decided to follow the [comment](https://github.com/ros-planning/navigation2/issues/3182#issuecomment-1397543983):
> Or, what if we set the namespace [here instead](https://github.com/ros-planning/navigation2/blob/main/nav2_bringup/launch/navigation_launch.py#L197)?

because of input parameter `constainer_name` will preserve its essence: relative container name. In this case, we do not need to worry about setting full container name relative to namespace as input parameter when launching `localization_launch.py`/`navigation_launch.py` separately without `bringup_launch.py`, which is more suitable for future utilization, I think.

Otherwise, if you like first approach, I will use it instead.